README.md

```diff
@@ -21,43 +21,35 @@ environment. It acts like an ether, an omnipresent, persistent layer
 for data. This way, application containers access their data
 regardless on which host the data nodes run.
 
-[Crate: Put your data to work. Simply.](https://crate.io/)
+[Crate](https://crate.io/)
 
-## Building Images
+![logo](https://raw.githubusercontent.com/docker-library/docs/2517900006ae5f4c03c1d43235930c59f4614394/crate/logo.png)
 
-Crate derives from
-[alpine:latest](https://hub.docker.com/_/alpine/). We build this
-nightly updated image from the `Dockerfile` on GitHub:
-
-    docker build -t="crate/crate" github.com/crate/docker-crate
-
-To build your own image, replace the image name after the `-t` option
-with your own name.
-
-Alternativly, an [Official Crate
-Image](https://hub.docker.com/_/crate/) with the name `crate` is
-available in the Docker Hub. That image is based on a stable major
-version of Crate, and is tested and scanned by Docker, Inc.
-
-## How to use this image
+## How To Use This Image
 
 To form a cluster, just start the Crate container a few times in the
-background. This starts a couple of containers on your machine which
-discover each other via multicast. In a production environment you'd
-run Crate on different machines:
+background. This starts a Crate container on your machine:
 
 ```console
 # docker run -d crate/crate
 ```
 
-To access the admin UI, point your browser to port tcp/4200 of a data
+To access the admin UI, map port 4200 and point your browser to port tcp/4200 of a
```

Comment:
Map from, to?
To access the admin UI you have two options: Either point your browser to port 4200 of the IP of a newly added node:
[...] --format - exmaple [...]
or map the port 4200 of the container to the same port number on the host. In latter case you may contact your UI at http://localhost:4200/admin. Please keep in mind that the second option is only possible once on each host, but since this is the recommended setup in most production environments, this is no restriction.